org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -33,7 +33,7 @@
         </location>
         <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
            <unit id="org.eclipse.lsp4j.sdk.feature.group" version="0.0.0"/>
```

Comment:
use 0.6.0 for the release. We should keep 0.0.0 for fast moving dependencies, i.e snapshot builds